sync.go

```diff
@@ -42,6 +42,10 @@ var (
 	disabled = func(args ...interface{}) string { return fmt.Sprint(args...) }
 )
 
+// Running too many syncs at once can exhaust file descriptor limits.
+// Empirically, ~90 is enough to hit the macOS default limit of 256.
+const maxConcurrentSyncs = 10
```

Comment:
If 90 is roughly the ceiling, how do you feel about something higher than 10 like 25?